db/dictionary-data/V0.2.0.600__impact-data.sql

```diff
@@ -0,0 +1,76 @@
+INSERT INTO
+    impact_scenario (name, description )
```

Comment:
be explicit regarding what columns get stuff inserted

```suggestion
    impact_scenario (id, name, description )
```

Reply:
:tired_face: Doh!  Good spot.  Updated now

Reply:
:tired_face: missed that one.  Silly copy&paste error.  Now fixed